plugins/factory-plugin/src/file-uri.ts

```diff
@@ -30,3 +30,19 @@ export default function convertToFileURI(file: string, rootFolder?: string): str
     return `file://${rootFolder}${file}`;
 
 }
+
+export function convertToCheProjectPath(fileURI: string, rootFolder: string): string {
+    if (rootFolder.endsWith('/')) {
+        rootFolder = rootFolder.substring(0, rootFolder.length - 1);
```

Comment:
Factory is backend plugin, so You could try to use  https://nodejs.org/docs/latest/api/path.html to simplify code
 `path.resolve(rootFolder, fileURI)`  and so on...
Import: `import * as path from 'path';

Reply:
looks like
```
    return `/${path.relative(rootFolder, fileURI)}`;
```
works